src/app/components/account/account-list-item-layout.tsx

```diff
@@ -37,17 +30,14 @@ export function AccountListItemLayout(props: AccountListItemLayoutProps) {
     accountName,
     avatar,
     balanceLabel,
+    showBtcAddress = false,
+    showStxAddress = false,
```

Comment:
Can we not infer this from whether or not a btc address is set? We could make that optional.

Reply:
hm, yeah, probably its better to remove this props at all and show address if it is provided